complete/src/main/java/dao/TourGuideDAO.java

```diff
@@ -0,0 +1,21 @@
+
+
+@Component
+public class TourGuideDAO implements ITourGuideDAO {
+
+    @Override
+    public TourGuide save(TourGuide tourGuide) {
+        return null;
+    }
+
+    @Override
+    public void delete(int id){
+
+    }
+
+    @Override
+    public List<TourGuide> fetchByGlobalSearch(String term) {
+        return null;
+    }
```

Comment:
Added a DAO for accessing a database

Source - https://www.tutorialspoint.com/java/java_data_structures.htm

Reply:
Ditto above.  Methods should do what their signatures say they do.  Having a method that returns null is dangerous, as a developer may see the method in an auto-complete, believe it is implemented, and then be surprised when it only returns null and causes NullPointerException.